utilities/src/apiSpecsList.json

```diff
@@ -36368,4 +36368,4 @@
       "2022-01-01-preview"
     ]
   }
-}
+}
```

Comment:
did this sneak in :D

Reply:
@ahmadabdalla - I can't get rid of it :D The contents is exactly the same as in main but GHUI showing a change

Reply:
Yeah - I think we can ignore this. I can only assume this comes down to the line ending or so...